### PR TITLE
Show overview notifications for cancelled and failed deployments

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1384419

openshift_web_console

@jwforres PTAL

### CODE REVIEW

File: app/scripts/directives/serviceGroupNotifications.js [hunk 1]
```diff
@@ -58,13 +60,55 @@ angular.module('openshiftConsole')
...
+                  rcLink = Navigate.resourceURL(mostRecentRC),
+                  logLink;
+              switch (status) {
+              case 'Cancelled':
+                alerts[mostRecentRC.metadata.uid + '-cancelled'] = {
+                  type: 'info',
+                  message: 'Deployment ' + displayName + ' was cancelled.',
+                  links: [{
+                    href: rcLink,
+                    label: 'View Deployment'
```
Comment (jwforres) [inline app/scripts/directives/serviceGroupNotifications.js]: should we have a second link to Redeploy or  "Start Deployment"

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**